src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,38 @@
+package duke.parser;
+
+/**
+ * Class for parsing user input.
+ */
+public class Parser {
+    public Parser() {
+    }
+
+    /**
+     * Gets a command from the user.
+     *
+     * @param input original String input from user
+     * @return an integer that represents a command
+     */
+    public int getCommand(String input) {
+        String[] i = input.split("\\s");
+        if (i[0].equals("bye")) {
+            return 0;
```

Comment:
Over here, number does not carry any meaning to a developer first time looking at your code. It would be better to have an Enum class or final static variables to grant these integer meaningful names.